Makefile

```diff
@@ -45,11 +45,17 @@ all build:
 #
 # Example:
 #   make build-tests
-build-tests:
-	hack/build-go.sh test/extended/extended.test
-	hack/build-go.sh test/integration/integration.test -tags='integration docker'
+build-tests: build-extended-test build-integration-test
 .PHONY: build-tests
 
+build-extended-test:
+	hack/build-go.sh test/extended/extended.test
+.PHONY: build-extended-test
+
+build-integration-test:
+	OS_GOFLAGS_TAGS='integration docker' hack/build-go.sh test/integration/integration.test
```

Comment:
It was hard-coded before.  The main purpose of the change here is to fix the breakage introduced in #12430.